SPECS/openssl-pkcs11/openssl-pkcs11.spec

```diff
@@ -1,46 +1,60 @@
+%define fedora 41
```

Comment:
This feels weird. I know we're pulling an updated spec from fedora, but we're not using it for fedora. It's also making the conditionals further down the file less intuitive. I look at them and think "Oh, 'if fedora', this isn't fedora, don't need to pay attention to that" which is not actually the case.

Reply:
+1, this is a bit confusing.

Reply:
Looking at it again, I agree. Removed all this nonsense.